vitest/runtime/runtime/samples/store-imported-module/foo.js

```diff
@@ -1,3 +1,3 @@
-import { writable } from '../../../../store';
+import { writable } from '../../../../../store';
```

Comment:
```suggestion
import { writable } from 'svelte/store';
```

I added a plugin to resolve these trough `svelte/store`. Although, it doesn't correctly get to this point because of the syntax error.

Reply:
Then again, custom loader probably means that hook doesn't apply. 🤔

Reply:
the plugin isn't necessary, it just resolves normally